### PR TITLE
IN-19 feat: add scrollbar

### PR DESCRIPTION
**Pull request by MaksimSinicbIn**

### CODE REVIEW

**Review by Abrosimov96 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -71,15 +71,16 @@
     "vite": "5.3.1"
   },
   "dependencies": {
-    "@radix-ui/react-dropdown-menu": "2.1.1",
     "@radix-ui/react-dialog": "1.1.1",
+    "@radix-ui/react-dropdown-menu": "2.1.1",
     "@radix-ui/react-popover": "1.1.1",
+    "@radix-ui/react-scroll-area": "^1.1.0",
```
Comment (vitaliiRss) [inline package.json:77]: Please remove ^ and rebuild pnpm-lock.yaml

File: src/components/scrollbar/scrollbar.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+const meta = {
+  argTypes: {
+    type: {
+      control: { type: "radio" },
+      options: ["auto", "always", "scroll", "hover"]
+    }
+  },
+  component: Scrollbar,
+  tags: ["autodocs"],
+  title: "01. Design System/Scrollbar"
```
Comment (SergFediaev) [inline src/components/scrollbar/scrollbar.stories.tsx:16]: title: "02. Components/Select"

File: package.json (hunk 1 above)
Comment (SergFediaev) [inline package.json:77]: "@radix-ui/react-scroll-area": "1.1.0",

File: src/components/scrollbar/scrollbar.module.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
+@use "../../styles/abstracts/index" as *;
+
+.root {
```
Comment (vitaliiRss) [inline src/components/scrollbar/scrollbar.module.scss:3]: please rename to "scrollableContainer"

File: src/components/scrollbar/scrollbar.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  ...restProps
+}: Props) => {
+  const maxHeightConverted = typeof maxHeight === "number" ? `${maxHeight}px` : maxHeight
+  const maxWidthConverted = typeof maxWidth === "number" ? `${maxWidth}px` : maxWidth
+
+  const viewportStyles = { maxHeight: maxHeightConverted, maxWidth: maxWidthConverted }
+
+  return (
+    <ScrollArea.Root asChild type={type}>
+      <div className={clsx(s.root, className)} {...restProps}>
```
Comment (vitaliiRss) [inline src/components/scrollbar/scrollbar.tsx:31]: Please move attributes className and restProps to ScrollArea element

File: src/components/scrollbar/scrollbar.stories.tsx (hunk 2 above)
Comment (web-serge) [reply src/components/scrollbar/scrollbar.stories.tsx]: fix

File: package.json (hunk 1 above)
Comment (web-serge) [reply package.json]: fix

File: src/components/scrollbar/scrollbar.module.scss (hunk 3 above)
Comment (web-serge) [reply src/components/scrollbar/scrollbar.module.scss]: fix

File: src/components/scrollbar/scrollbar.tsx (hunk 4 above)
Comment (web-serge) [reply src/components/scrollbar/scrollbar.tsx]: fix

**Review by vitaliiRss — APPROVED** [review]

**Review by SergFediaev — APPROVED** [review]

**Pull request merged**